bin/list_root_schemas

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
Could add it into a script called `gencode_doc_utils` along with other gendoc related scripts to reduce the number of those?  I don't think `gencode_schemas` is a good name for this particular script.

Incidentally .. I was planning to split the HTML documentation generation out of gendocs and calling **that** `gencode_schemas` because `gencode_docs` itself is getting messy to work with